spec/features/movies/index_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe 'as a user on the movies index page' do
```

Comment:
Same change as in the discover_spec: add "registered" before user

Reply:
You got it!